### ISSUE TITLE
SslNegotiateCert in web.config results in 'cannot determine process id DNX'

### ISSUE THREAD

**Issue by jochenvanwylick** [aspnet/Home#1095]
I'm trying to enable client certificate auth in my vNext ( rc-1-final ) web app with WebApi.

After doing `file > new project` for my web app - and running the app - the `/api/values` controller returns the expected JSON. 

However - as soon as I add:

```
<security>
    <access sslFlags="SslNegotiateCert"/>
</security>
```

To the web.config - I get the following message upon running the web app:

> An error occurred attempting to determine the process id of the DNX process hosting your application

What am I doing wrong?

**Comment by Eilon** [comment]
@blowdart @Tratcher thoughts?

BTW what section is this in `web.config`? Please note that `web.config` is used only for IIS settings and ASP.NET 4.x. It is not used by ASP.NET 5 at all.

**Comment by jochenvanwylick** [comment]
@Eilon this is the `<system.webServer>` section. If it's not used - is there support for client certs in vNext then?

**Comment by blowdart** [comment]
Not *yet*

**Comment by Tratcher** [comment]
@BillHiebert This looks like the same issue as you hit when anonymous auth is disabled. That said, I would only expect this failure with SslRequireCert, not SslNegotiateCert.

@jochenvanwylick this should only fail with F5 (debug), not CTL+F5.

**Comment by BillHiebert** [comment]
Sounds like it. I opened a bug in our tooling database for this.

**Comment by jochenvanwylick** [comment]
OK guys - thanks!

**Comment by sayedihashimi** [comment]
TFS ID: 166457

**Comment by BillHiebert** [comment]
Note that to get this to work you need to unblock delegation of security access in the applicationhost.config file. Open

**Comment by BillHiebert** [comment]
@jochenvanwylick

**Comment by jochenvw** [comment]
@BillHiebert Hi Bill - thanks for the heads-up, I've left the company in the meantime to join you guys. We're colleagues now :). However - I will pass this on the team.

**Comment by aspnet-hello** [comment]
This issue is being closed because it has not been updated in 3 months.

We apologize if this causes any inconvenience. We ask that if you are still encountering this issue, please log a new issue with updated information and we will investigate.
